src/bitte-wallet.ts

```diff
@@ -191,7 +191,7 @@ const signUsingKeyPair = async (
 
   return account.signAndSendTransaction({
     receiverId,
-    actions: actions.map((a) => createAction(a)),
+    actions: actions.map((a) => internalActionToNaj(a as any)),
```

Comment:
In v10, the `createAction` function was removed and replaced with `internalActionToNaj`. this function essentially performed the same task: returning an action that is compatible with naj